docs/changelog/4.2.rst

```diff
@@ -0,0 +1,1147 @@
+Changelogs for 4.2.x
+====================
+
+.. changelog::
+  :version: 4.2.0
+  :released: *unreleased*
+
+  This release *blah blah blah* …
+
+  Major features for this release:
+
+  * Lua records
+  * IXFRDist
+  * API changes
+  * DNSSEC? (#6655)
```

Comment:
I dont think #6655 is a major feature (it is a bug fix)

Reply:
Ok thanks, I wasn't sure if it was something important that people might want to know about.